tools/profile_endpoints.py

```diff
@@ -274,7 +278,44 @@ def profile_get_concepts(token):
 
 # Main function
 if __name__ == "__main__":
-    authenticate()
+
+    # Create the parser
+    parser = argparse.ArgumentParser(
+        description="Profile endpoints for a specific host and dataset"
+    )
+
+    # Add the arguments
+    parser.add_argument(
+        "--host",
+        type=str,
+        choices=HOSTS.keys(),
+        default="local",
+        help="The host to profile",
+    )
+
+    parser.add_argument(
+        "--dataset",
+        type=str,
+        choices=DATASETS.keys(),
+        default="gcp",
+        help="The dataset to profile",
+    )
+
+    parser.add_argument(
+        "--authenticate",
+        type=bool,
+        default=False,
+        help="If false, use current gcloud authentication. If true, authenticate with "
```

Comment:
Will this auth you back if you set auth to true?

Reply:
If you set it to true, it will bring up the google authn page in a web browser, so you can login as a different user.

Maybe a fancier version would let you switch to a different user that's already logged in vs running authn in the browser each time. For me, I was always running as the same user so I didn't want to login in the browser each time.